packages/sources/ncfx/src/endpoint/crypto.ts

```diff
@@ -93,6 +132,10 @@ export const cryptoTransport = new WebSocketTransport<EndpointTypes>({
   },
 })
 
+const isInfoMessage = (message: WsMessage): message is WsInfoMessage => {
+  return (message as WsInfoMessage).Type !== undefined
```

Comment:
Any way to make this a more specific check, like `['Type1', 'Type2'].includes(Type)`?

Reply:
So far the only `Type` value I've seen is `Info`, but I don't know if there could be others (e.g. other log levels or other values entirely) and there's no API docs to reference :(. The info type messages look fairly different from the price type messages so I feel like this is sufficient?